docs/alerts/alert-rules.md

```diff
@@ -0,0 +1,163 @@
+
+#### Transformation
+
+Transformations modify metrics using mathematical expressions and functions. For instance, if you want to be alerted when both `request.size` and `response.size` exceed a threshold or become anomalous, you can transform them into a single data series using an expression like:
+
+```
+request.size + response.size
+```
+
+You can then create an alert based on this new data series. For more information on transformations, visit [Transformations](../dashboards/chart-builder/#transformation).
+
+#### Filter by
+
+The **Filter** section [defines criteria](https://sematext.com/docs/dashboards/chart-builder/#filter-by) that will trigger an alert only under specific conditions. For example, if you collect data from multiple hosts and want to create an alert for a specific host, choose the tag name `os.host` and set the value to `host-x`.
+
+#### Group by
+
+In the **Group by** section, select what [data series to group on](https://sematext.com/docs/dashboards/chart-builder/#group-by). For example, choosing the `os.host` tag and setting the aggregation in the **Aggregation** field to `all separately` means the alert will trigger for each host individually.
+
+#### Rollup by
+
+Data points sent to Sematext Cloud have a time label. If a metric is updated every 10 seconds and you’re looking at a 24-hour time range, you need 14,400 points to display all data points. The **Rollup by** function groups data points into time buckets. Metrics coming in within the alert's specified time window will be aggregated by the selected function in this section. Refer to [Rollup By](../dashboards/chart-builder/#rollup-by) for more details.
+
+### Alert Condition
+
+Here, you will select the [alert type](https://sematext.com/docs/alerts/#alert-types) and priority. If you select a threshold type alert, enter the threshold value required to trigger an alert and specify a time window.
+
+#### Ignore Regularly Occurring Spikes and Dips
+
+Historical data is analyzed to determine if an alert fits the regular stream of spikes and dips in metrics. If it does, notifications will not be sent. The filter combination is used to identify previous spikes and dips, grouping them into similar values (±5% assumed to be the same). The least common multiple (LCM) in these groups is calculated to check if the current value fits any of them.
+
+### Meta Data
+
+The **Meta Data** tab allows you to enter details about the alert, such as the alert name, description, and runbook.
+
+#### Runbook
+
+The runbook serves as a detailed guide outlining procedures, recommended actions, and best practices to follow when an alert is triggered. It ensures a standardized and efficient approach to managing routine operations, handling system updates, addressing issues, and promptly restoring services upon alert activation.
```

Comment:
We have a pretty solid Glossary page/guide for Runbooks.... if we want to link to that?